docs/02-guides/configuration.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: Configurations
```

Comment:
**nitpick:** guides being task oriented, I think the title should also be.

**suggestion:** rename to "Inject configurations", "Customize configurations" or "Override configurations"